src/Parser.cpp

```diff
@@ -0,0 +1,69 @@
+#include "Parser.h"
+std::vector<Price_info> Parser::load (std::string data){
+
+    for (std::size_t i=0; i<=data.length(); i++){
+        if(data[i] == '"' or data[i] == '[' or data[i] == '{' or data[i] == ',' or data[i] == ':' or data[i] == '}'|| data[i] == ']' ){
+            data[i] = ' ';
+        }
+        if(data[i]=='s' and data[i+1]=='y' and data[i+2]=='m' and data[i+3]=='b' and data[i+4]=='o' and data[i+5]=='l'){
```

Comment:
here and in above/below cases code is not very readable
```
        if(data[i]=='s' and data[i+1]=='y' and data[i+2]=='m' and data[i+3]=='b' and data[i+4]=='o' and data[i+5]=='l'){
            data[i]=' ';
            data[i+1]=' ';
            data[i+2]=' ';
            data[i+3]=' '; 
            data[i+4]=' ';
            data[i+5]=' ';
        }
```

should be converted to 
```
        if(isSymbolString(data,i)){
           std::string symbol_str("symbol");
           fillWithSpaces(data,i, str.length())
        }
```

same for rest of if cases

Reply:
resolved